x-pack/plugins/security_solution/public/cases/components/add_comment/index.test.tsx

```diff
@@ -153,6 +155,10 @@ describe('AddComment ', () => {
       </TestProviders>
     );
 
+    act(() => {
```

Comment:
You do not need the `act` here. Just drop the call to `attachTimeline` down inside of the `waitFor`
```
    await waitFor(() => {
      attachTimeline('[title](url)');
      expect(wrapper.find(`[data-test-subj="add-comment"] textarea`).text()).toBe('[title](url)');
    });
```